source/Ship.h

```diff
@@ -354,10 +354,14 @@ class Ship : public Body, public std::enable_shared_from_this<Ship> {
 	double Mass() const;
 	double InertialMass() const;
 	double TurnRate() const;
-	double Acceleration() const;
-	double MaxVelocity() const;
+	double ForwardThrust(bool allThrust) const;
+	double ForwardAcceleration(bool allThrust) const;
```

Comment:
```suggestion
	double ForwardThrust(bool allThrust = false) const;
	double ForwardAcceleration(bool allThrust = false) const;
```

Reply:
I initially wanted to implement it as you suggested as using an afterburner clearly should be the exception when having a regular thruster. Then I hesitated and wanted to check the users of that function first.
While doing so I learned two things:

- There is value in being explicitly forced to think about whether in a specific situation an additional afterburner should be used
- Although most of the (in-game) time an additional afterburner won't be used, the majority of AI logic might still use an afterburner as the special cases typically include some urgency

Do you still think, that I should commit the change proposed by you?

Reply:
same here, I think it would be the right way, but I dont want you to do something only to revert it later so lets wait on another opinion